src/app/dashboard/settings/booking-events/_actions.ts

```diff
@@ -62,3 +62,35 @@ export default async function createEventType(
   permalink && revalidatePath(permalink);
   return { success: `Event type '${res.data.title}' created successfully.` };
 }
+
+export async function deleteEventType(
+  _prevState: { error: null | string } | { success: null | string },
+  eventTypeId: number
+) {
+  const sesh = await auth();
+  if (!sesh?.user.id) {
+    console.log("[_actions] Unauthorized user delete");
```

Comment:
Either remove this or change this to a console.error

Reply:
Replaced with console.error in [refactor: console.log -> console.error](https://github.com/calcom/platform-starter-kit/pull/46/commits/b7ae5ac15bc3260e60439d8aac3d5c0604ffe057)